extensions/wikia/Search/classes/IndexService/DefaultContent.php

```diff
@@ -83,7 +83,8 @@ public function execute() {
 				'page_images'                => count( $response['parse']['images'] ),
 				'iscontent'                  => $service->isPageIdContent( $pageId ) ? 'true' : 'false',
 				'is_main_page'               => $service->isPageIdMainPage( $pageId ) ? 'true' : 'false',
-				];
+				'indexed'                    => gmdate("Y-m-d\TH:i:s\Z")
```

Comment:
LGTM, but...
Unit test for this file belongs to "Broken" group so it is not being checked during run. Do we need test for this anyway?